tests/utils/server_manager.py

```diff
@@ -67,25 +82,35 @@ def start(self):
                 )
 
     def start_consul(self):
+        try:
+            requests.get('http://127.0.0.1:8500/v1/catalog/nodes')
+        except Exception:
+            pass
```

Comment:
nit: Also would like to see logging of this case as well.

Reply:
Similar to the previous, the error message is displayed via thrown exception in the "else" clause of the try..catch..else block.